packages/api/legacy-require-handler.cjs

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-commonjs */
+
+/**
+ * You can regenerate the type declarations for this file by running the following:
+ * ```sh
+ * npx tsc legacy-require-handler.cjs --checkJs --declaration --emitDeclarationOnly
```

Comment:
want to add this as `build:legacy-handler` script that's run post `npm run build`?